lib/apicontracts.js

```diff
@@ -4827,18 +5099,12 @@ class CreditCardType extends CreditCardSimpleType {
 			if(('cardCode' in obj) && (obj.cardCode != null)) { this.setCardCode(obj.cardCode); }
 			if(('isPaymentToken' in obj) && (obj.isPaymentToken != null)) { this.setIsPaymentToken(obj.isPaymentToken); }
 			if(('cryptogram' in obj) && (obj.cryptogram != null)) { this.setCryptogram(obj.cryptogram); }
-			if(('tokenRequestorName' in obj) && (obj.tokenRequestorName != null)) { this.setTokenRequestorName(obj.tokenRequestorName); }
-			if(('tokenRequestorId' in obj) && (obj.tokenRequestorId != null)) { this.setTokenRequestorId(obj.tokenRequestorId); }
-			if(('tokenRequestorEci' in obj) && (obj.tokenRequestorEci != null)) { this.setTokenRequestorEci(obj.tokenRequestorEci); }
```

Comment:
Why were these lines removed?

Reply:
These lines were added for ChasePay sdk changes. Did you make sure your fork was updated before you made a commit and push?